File: biometric/src/main/java/dev/skomlach/biometric/compat/engine/BiometricInitListener.java
package dev.skomlach.biometric.compat.engine;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.annotation.RestrictTo;

import dev.skomlach.biometric.compat.engine.internal.core.interfaces.BiometricModule;

@RestrictTo(RestrictTo.Scope.LIBRARY)
public interface BiometricInitListener {
    void initFinished(@NonNull BiometricMethod method, @Nullable BiometricModule module);

    void onBiometricReady();
}

File: biometric/src/main/java/dev/skomlach/biometric/compat/engine/BiometricInitListener.kt
package dev.skomlach.biometric.compat.engine

import androidx.annotation.RestrictTo
import dev.skomlach.biometric.compat.engine.internal.core.interfaces.BiometricModule

@RestrictTo(RestrictTo.Scope.LIBRARY)
interface BiometricInitListener {
    fun initFinished(method: BiometricMethod, module: BiometricModule?)
    fun onBiometricReady()
}